### PR TITLE
docs: Community Meeting Minutes - Jan 4, 2024

### PR DESCRIPTION
**Pull request by Billy99**

### CODE REVIEW

File: docs/blog/posts/community/2024/2024-01-04-meeting.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Some of the next work items for `bpfman` revolve around removing the async code
+from the code base, make `bpfman-core` a rust library, and removing all the
+gRPC logic.
+Dave is currently investigating this.
+One area to help out is to take the `bpfman-csi` thread and making it it's own
+binary.
+This may require making `bpfman` a bin and lib crate (which is fine, just needs
+a lib.rs and to be very careful about what we’re exporting).
+Andrew is starting to take a look at this.
```
Comment (astoycos) [inline docs/blog/posts/community/2024/2024-01-04-meeting.md:36]: Github tags would be better here but this is fine for now 👍

**Review by astoycos — APPROVED** [review]
Just one nit but I'm alright with it, thanks for the work @Billy99!

**Comment by astoycos** [comment]
(added hold if others want to take a look please remove when ready @Billy99 )

File: docs/blog/posts/community/2024/2024-01-04-meeting.md (hunk 1 above)
Comment (Billy99) [reply docs/blog/posts/community/2024/2024-01-04-meeting.md]: Done

**Review by anfredette — APPROVED** [review]
LGTM!  Thanks for putting this together, Billy!

**Pull request merged**